_data/fellows.yml

```diff
@@ -1,11 +1,11 @@
 gabrielleBeaudry:
   name: Gabrielle Beaudry
-  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  info: 'BS Environmental Science, Energy Management track; Minor in Computer Science at UNC Chapel Hill'
   year: 2018
   image: gabrielleBeaudry.jpg
 xiaoWu:
   name: Xiao Wu
-  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  info: 'Master of Urban Spatial Analytics and Landscape Architecture and Regional Planning at the University of Pennsylvania School of Design'
```

Comment:
candidate: 
'Master of Urban Spatial Analytics and Landscape Architecture and Regional Planning Candidate at the University of Pennsylvania School of Design'